_data/sites.json

```
@@ -1718,6 +1718,17 @@
             "support.beamng.com"
         ]
     },
+    
+    {
+        "name": "Beatport",
+        "url": "https://support.beatport.com/hc/en-us/articles/4412533928596-How-can-I-close-my-account-",
+        "difficulty": "hard",
+        "email": "support@beatport.com",
```

```suggestion
        "email": "support@beatport.com",
        "notes": "Account deletion requires contacting Customer Support.",
        "notes_tr": "Hesabınızın silinmesi için müşteri hizmetleri ile iletişime geçmeniz gerekir.",
        "notes_fr": "Pour supprimer votre compte contactez le support.",
        "notes_it": "La cancellazione account richiede di contattare il servizio clienti.",
        "notes_de": "Account-Löschung erfordert es den Kundensupport zu kontaktieren",
        "notes_pt_br": "Para deletar a sua conta contate a assistência ao cliente",
        "notes_cat": "Per esborrar el seu compte contacti amb l'assistència al client",
        "notes_es": "Para borrar tu cuenta contacta con asistencia al cliente",
```

I suggest adding some notes in different languages.